cnxpublishing/subscribers.py

```diff
@@ -118,6 +118,18 @@ def baking_processor(self, module_ident, ident_hash, cursor=None):
 
     logger.debug('Starting baking module_ident={} ident_hash={}'
                  .format(module_ident, ident_hash))
+
+    recipe_ids = _get_recipe_ids(module_ident, cursor)
+
+    state = 'current'
+    if recipe_ids == (None, None):
+        remove_baked(ident_hash, cursor=cursor)
+        logger.debug('Finished unbaking module_ident={} ident_hash={} '
+                     'with a final state of \'{}\'.'
+                     .format(module_ident, ident_hash, state))
+        update_module_state(cursor, module_ident, state, None)
+        return
```

Comment:
Why don't we just have a script to remove the baked content? That's what this logical path is about, no?

I'm just not sure this is the place to put this. It feel like, "(spoken to the babysitter just before walking out the door) O... yeah, and don't let the baby play with the kitchen knives." In other words, it feels like a special exception rather than something that is going to be conditionally valid on a regular basis.

Reply:
Actually this will be the primary path for all non-openstax books until we create the generic-default-recipe, which is at least 6 months  (1 year?) down the road.

Reply:
IN the light of morning, it seems so much clearer and obvious. Test pushed.

Reply:
I guess a deeper test would have baked it, then unbaked it  and verified that it went away. This tests that the no recipe path is used at all.